central/processlisteningonport/store/postgres/full_store.go

```diff
@@ -46,6 +46,16 @@ const getByDeploymentStmt = "SELECT plop.serialized, " +
 
 const getClusterAndNamespaceStmt = "SELECT namespace, clusterid FROM deployments WHERE id = $1"
 
+// const getCountsByDeploymentId = "SELECT deploymentid, namespace, clusterid, count(*) as count FROM listening_endpoints where closed = false GROUP BY deploymentid, namespace, clusterid"
+const getCountsByDeploymentId = "SELECT d.id, d.namespace, d.clusterid, COALESCE(COUNT(le.deploymentid), 0) AS count " +
```

Comment:
Do you have a benchmark and/or explain plan for this?

Reply:
I have included an explain plan in the PR description. I am still planning on doing load testing with k6.